assembly/gradle-plugin/src/test/resources/pom.xml

```diff
@@ -80,14 +81,6 @@
                         <phase>package</phase>
                     </execution>
                 </executions>
-                <dependencies>
```

Comment:
Why is this not required anymore?

Reply:
This dependency is not existing anymore, the functionality has been integrated by Onur into the core workflow steps.